pkg/config/remote/service/service.go

```diff
@@ -196,6 +203,13 @@ func NewService() (*Service, error) {
 	}, nil
 }
 
+func newOrgUUIDProvider(ctx context.Context, http api.API) uptane.OrgUUIDProvider {
```

Comment:
nit: It's not just any OrgUUIDProvider it's `newRCBackendOrgUUIDProvider` (or something along those lines, naming things is hard. 😁 )